src/Bridge/Doctrine/Orm/Filter/OrderFilter.php

```diff
@@ -171,4 +174,38 @@ protected function extractProperties(Request $request/*, string $resourceClass*/
 
         return \is_array($properties) ? $properties : [];
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function addJoinsForNestedProperty(string $property, string $rootAlias, QueryBuilder $queryBuilder, QueryNameGeneratorInterface $queryNameGenerator/*, string $resourceClass*/): array
+    {
+        if (\func_num_args() > 4) {
+            $resourceClass = func_get_arg(4);
+        } else {
+            if (__CLASS__ !== \get_class($this)) {
+                $r = new \ReflectionMethod($this, __FUNCTION__);
+                if (__CLASS__ !== $r->getDeclaringClass()->getName()) {
+                    @trigger_error(sprintf('Method %s() will have a fifth `$resourceClass` argument in version API Platform 3.0. Not defining it is deprecated since API Platform 2.1.', __FUNCTION__), E_USER_DEPRECATED);
+                }
+            }
+            $resourceClass = null;
+        }
+
+        $propertyParts = $this->splitPropertyParts($property, $resourceClass);
+        $parentAlias = $rootAlias;
+        $alias = null;
+
+        foreach ($propertyParts['associations'] as $association) {
+            // Force left joins on ordering because we don't want to filter
```

Comment:
Hmm... Is it possible to add another argument to the function instead of duplicating it here? (I know it already suffers from too many arguments...) :rofl:

Reply:
The `AbstractFilter` method... How will this be different in 3.0?

Reply:
because we would remove a lot of code for this 4th argument :p. 

Anyway because it could be necessary in other filters I'll add a 5th one!